### PR TITLE
[frontend] add translations

### PR DESCRIPTION
**Pull request by savacano28**
### Proposed changes

*
*

### Related issues

* https://github.com/OpenBAS-Platform/openbas/issues/1231
*

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [x] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: openbas-front/src/utils/Localization.js [hunk 1]
```diff
@@ -543,6 +549,8 @@ const i18n = {
         'Souhaitez-vous supprimer cette pression médiatique ?',
       'Do you want to delete this challenge?':
         'Souhaitez-vous supprimer ce challenge ?',
+      'Used challenges (in injects)': 'Défis utilisés (dans les injects)',
```
Comment (Dimfacion) [inline openbas-front/src/utils/Localization.js:552]:
```suggestion
      'Used challenges (in injects)': 'Défis utilisés (dans les stimulis)',
```

File: openbas-front/src/utils/Localization.js [hunk 2]
```diff
@@ -543,6 +549,8 @@ const i18n = {
...
+      'No challenge are used in the injects of this simulation.': 'Aucun défi n\'est utilisé dans les injects de cette simulation.',
```
Comment (Dimfacion) [inline openbas-front/src/utils/Localization.js:553]:
```suggestion
      'No challenge are used in the injects of this simulation.': 'Aucun défi n\'est utilisé dans les stimulis de cette simulation.',
```

File: openbas-front/src/utils/Localization.js [hunk 3]
```diff
@@ -920,6 +932,7 @@ const i18n = {
       Agents: 'Agents',
       'Learn more information about how to setup simulation agents': 'En savoir plus sur la configuration des agents de simulation',
       'in the documentation': 'dans la documentation',
+      'Here, you can download and install simulation agents available in your executors. Depending on the integrations you have enabled, some of them may be unavailable.': 'Voici, vous pouvez télécharger et installer les agents de simulation disponibles pour vos exécuteurs. Selon les intégrations que vous avez activées, certains agents peuvent ne pas être disponibles.',
```
Comment (Dimfacion) [inline openbas-front/src/utils/Localization.js:935]:
```suggestion
      'Here, you can download and install simulation agents available in your executors. Depending on the integrations you have enabled, some of them may be unavailable.': 'Vous pouvez ici télécharger et installer les agents de simulation disponibles pour vos exécuteurs. Selon les intégrations que vous avez activées, certains agents peuvent ne pas être disponibles.',
```

File: openbas-front/src/utils/Localization.js [hunk 4]
```diff
@@ -1165,46 +1180,47 @@ const i18n = {
...
+      'Content should not be empty': 'Le contenu ne doit pas être vide',
+      Timezone: 'Fuseau horaire',
+      Log: 'Journal',
+      'Your file should be a XLS': 'Votre fichier doit être un XLS',
+      'Launch import': "Lancer l'importation",
+      'Show timeline': 'Afficher la chronologie',
+      'Selected payload': 'Charge utile sélectionnée',
+      Prevention: 'Prévention',
+      'Start date should be at least today': "La date de début doit être au moins aujourd'hui",
+      'Update the xls mapper': 'Mettre à jour le mapper xls',
```
Comment (Dimfacion) [inline openbas-front/src/utils/Localization.js:1205]: I know it's not linked to your PR but :
```suggestion
      'Update the xls mapper': 'Mettre à jour le mappeur xls',
```

File: openbas-front/src/utils/Localization.js (hunk 3 above)
Comment (savacano28) [reply openbas-front/src/utils/Localization.js:935]: Ty!!

**Review by Dimfacion — APPROVED** [review]

**Pull request merged**